docs/config-spec.md

```diff
@@ -17,73 +17,68 @@ For example:
 # Metadata
 #####################
 
-# TBD
-# Unique identifier for this observability pack
-# uuid: guid, required
+# A unique Observability Pack identifier
+uuid: guid, required
+
+# The name of the pack displayed everywhere the pack is referenced
+title: string, optional
 
 # Name acts as a unique identifier for this observability pack
 # Formatted as hyphenated lowercase
 # Example: apache, golden-signals-for-web-servers
 name: string, required
 
-# Displayed on the pack overview/details page formatted into rich text.
-# Standardized to a template for the New Relic packs.
+# A long form description for this Observability Pack.
 description: string, required
 
-# TBD
-# url slug for linking on public website
-# recommended format: make this a lowercase and hyphenated version
-# of the `name` field
-# slug: string, required
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+# A short form description for this Observability Pack.
+summary: string, required
```

Comment:
This is not currently required in the json schema

Reply:
This is currently listed as a non-nullable field in graphql

<img width="367" alt="Screen Shot 2021-08-19 at 8 50 46 AM" src="https://user-images.githubusercontent.com/1395158/130090646-e55361ee-99fd-412a-8898-945f9009e97b.png">